.github/pull_request_template.md

```diff
@@ -1,25 +1,37 @@
-# Description
+## Description
 
-Please include a summary of the change.
+*Replace this paragraph with a description of what this PR is doing. If you're modifying existing behavior, describe the existing behavior, how this MR is changing it, and what motivated the change. If this is a breaking change, specify explicitly which APIs have been changed*
 
-Fixes # (issue number, if there is one)
+## Checklist
 
-## Type of change
+Before you create this PR confirm that it meets all requirements listed below by checking the relevant checkboxes (`[x]`). This will ensure a smooth and quick review process.
 
-Please delete options that are not relevant.
+- [ ] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
```

Comment:
Too many boxes! The cats would love it though

Reply:
Miaauw :cat: